app/models/intervention.rb

```diff
@@ -4,7 +4,7 @@ class Intervention < ActiveRecord::Base
   belongs_to :intervention_type
   belongs_to :school_year
   has_many :progress_notes, dependent: :destroy
-  validates_presence_of :student_id, :intervention_type_id, :start_date
+  validates :student, :intervention_type, :start_date, presence: true
```

Comment:
In testing the JS error rendering, I noticed that validations like like only guarded for the `intervention_type_id` field having a value, but not that the value was a valid foreign key.  Maybe there's a better Rails-ism to do this, but a quick Googling led to changing this to use `validates` on the association name, and that worked to guard for invalid values as well.